### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by Wayneaswell** [KillerInk/FreeDcam#173] [Ideas]
I use freedcam for custom microscopy, and have an interest in pro video and quality video.  Please if the following could be considered.  Thank you.

- Customised profile launch icons (straight into customised setup for different shooting).

- Feature Request menu item. :)

- Ability to shoot stills at full sensor resolution. (48mp etc).  Even if just by remosaic reconstruction, when nothing is available.

- Help menu describing all options for novices to learn.

- custom codec addition.

- Pro video dng in virtual container (to avoid file operations slow down from saving frames as individual files).  With meta data, with pixels marching the underlying sensor mosaic pattern, from which raw data can be reconstructed afterwards.  Like in black magic Raw (BRAW).  Use of heif.

- ProRes codec support (there is an open sourced clone).

- Paid pro video version of app. $10.  Customisable GUI and external user controls through wireless and USB.

Thank you.

**Comment by KillerInk** [comment]
> Customised profile launch icons (straight into customised setup for different shooting).

should be relative easy to realize with programable widgets

> Feature Request menu item. :)

Im a lazy guy. makes it then to easy to get in contact^^

> Ability to shoot stills at full sensor resolution. (48mp etc). Even if just by remosaic reconstruction, when nothing is available.

on some phones the binning can get turned off. but on most with qcfa pattern not.
think a problem could get the memory usage and its alot work to implement. 
not sure if that will ever happen and if its the work worth, because offical api2 have it with next android version.

> Help menu describing all options for novices to learn.

feel free to contribut and extend the wiki with the stuff you are figuring out^^

> custom codec addition.

>Pro video dng in virtual container (to avoid file operations slow down from saving frames as individual files). With meta data, with pixels marching the underlying sensor mosaic pattern, from which raw data can be reconstructed afterwards. Like in black magic Raw (BRAW). Use of heif.

>ProRes codec support (there is an open sourced clone).


there is a video profile editor, with that you can select the supported codecs that are avalible on your phone.
mostly h264 and hevc on current gen phones. using other codecs without hw support can reduce performance.
the best you can get currently is 10bit


>Paid pro video version of app. $10. Customisable GUI and external user controls through wireless and USB.

freedcam is opensource. for me this is a hobby. i work on it when i have time

about remote control
#103 #105

**Reply by Wayneaswell** [reply]
Well thanks for answering.  I've had a brain disease with permanent damage, am trying to rebuild my life with 100x too much.  I have done 100x too much helping for a sick person.

Bugs. Usually crashes on record, old photos shown when.dlidomg sccrisdy,cam crash.  Moto 8+ android 10, stick at version 39.17 due to will not install error using either this or fdroid since last year.

The onboard codecs are weak supported by weak hardware, like most.  But, a software codec like the free version of conform, is very processor economical, and the only way to improve things, apart from software intra codec.

If I knew what things were to put in a wiki, I wouldn't be asking what things where.  :)

Well thanks if next android will have 48mp support.  I'll have to swap to a phone that gives that android.

Where are the programmable widgets?

Thank you.

**Comment by Wayneaswell** [comment]
Bug report.

I got to admit, Imiae.it only for microscopes because it was so buggy.  But now it seems really stable.

I was playing around with the settings and got it recording over 120mb/s h264 and about the sand HECV, which brings a s.e to my heart.  Unfortunately I have been so sick, I no longer have a computer set up to drag on the footage, and see if it's gappeng.for real.  But both codecs report MP4 file extended.  I find see this midrange motog8+
 If true on the hevc. That's pretty good.

Bugs, setting a profile to 8k has introduced an destabilising  permanent "preparation" error, even when going back to 4k.

Upon fresh install the first time, auto exposure was burnt out .

The preview thumbnail of the video dies not show up.

Is there a way to put all the controls away from the pop-up android home control keys?  They are good there, but trying to get out of the camera app results on in it just sliding the preview window over nearly a the time.

But, thanks for this again.
